translations/harbour-counter-pl.ts

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pl">
+<context>
+    <name></name>
+    <message id="counter-app_name">
+        <source>Counter</source>
+        <extracomment>Application title</extracomment>
+        <translation>Counter</translation>
+    </message>
+    <message id="counter-menu-delete_counter">
+        <source>Delete this counter</source>
+        <extracomment>Pulley menu item</extracomment>
+        <translation>Usuń ten licznik</translation>
+    </message>
+    <message id="counter-menu-new_counter">
+        <source>New counter</source>
+        <extracomment>Pulley menu item</extracomment>
+        <translation>Nowy licznik</translation>
+    </message>
+    <message id="counter-remorse-delete_counter">
+        <source>Deleting this counter</source>
+        <extracomment>Remorse popup text</extracomment>
+        <translation>Usuwanie tego licznika</translation>
+    </message>
+    <message id="counter-default_title">
+        <source>Counter</source>
+        <extracomment>Default title for a new counter</extracomment>
+        <translation>Licznik</translation>
+    </message>
+    <message id="counter-title">
+        <source>Label</source>
+        <extracomment>Text field label (counter title)</extracomment>
+        <translation>Etykieta</translation>
+    </message>
+    <message id="counter-title-placeholder">
+        <source>Short description of this counter</source>
+        <extracomment>Text field placeholder (counter title)</extracomment>
+        <translation>Krótki opis tego licznika</translation>
+    </message>
+    <message id="counter-switch-favorite">
+        <source>Show on cover</source>
+        <extracomment>Text switch label</extracomment>
+        <translation>Pokaż na etykiecie</translation>
+    </message>
+    <message id="counter-switch-favorite-description">
+        <source>No more than two counters can be shown on cover at the same time.</source>
+        <extracomment>Text switch description</extracomment>
+        <translation>Nie można jednocześnie wyświetlać więcej niż dwóch liczników na okładce.</translation>
+    </message>
+    <message id="counter-switch-sounds">
+        <source>Play sounds</source>
+        <extracomment>Text switch label</extracomment>
+        <translation>Odtwarzaj dźwięki</translation>
+    </message>
+    <message id="counter-button-reset">
+        <source>Reset</source>
+        <extracomment>Button label (resets counter to zeto)</extracomment>
+        <translation>Resetuj licznik</translation>
```

Comment:
How about just "Resetuj" to shorten the button text?

Reply:
To be honest, just "Resetuj" is not good. "Reset" or "Resetuj licznik" is ok.

Reply:
OK. I'll have to trust you on this one :)